store/localstore/local_region.go

```diff
@@ -0,0 +1,318 @@
+package localstore
+
+import (
+	"bytes"
+
+	"encoding/binary"
+	"github.com/golang/protobuf/proto"
+	"github.com/juju/errors"
+	"github.com/pingcap/tidb/kv"
+	"github.com/pingcap/tidb/terror"
+	"github.com/pingcap/tidb/util/codec"
+	"github.com/pingcap/tidb/util/types"
+	"github.com/pingcap/tidb/xapi/tablecodec"
+	"github.com/pingcap/tidb/xapi/tipb"
+)
+
+// local region server.
+type localRegion struct {
+	id       int
+	store    *dbStore
```

Comment:
When ChangeRegionInfo is called, do we need to migrate data from other localRegion's store?